### ISSUE TITLE
COG uses 50% cpu

### ISSUE THREAD

**Issue by ghost** [beaufortfrancois/cog-chrome-app#5]
As discussed in the thread on Google+  launching COG drives CPU usage up from 4% to about 60%
![clean start no cog](https://cloud.githubusercontent.com/assets/8536767/4023340/d93b10f6-2b87-11e4-9010-f28dc1e20260.png)
![cog running](https://cloud.githubusercontent.com/assets/8536767/4023341/dbd8c010-2b87-11e4-9a3e-a0ca5ab53d12.png)

**Comment by jgw96** [comment]
Same problem on my Acer c720 chromebook; cog uses about 1.85 gigs of memory. Could the chrome system api's somehow hog memory?

**Comment by vivekannan** [comment]
Not sure if these changes would help with the CPU usage but we could try,
1. Use chrome API event listeners for [storage](https://developer.chrome.com/apps/system_storage#event-onAttached) and [display](https://developer.chrome.com/apps/system_display#event-onDisplayChanged).
2. Should query for the selectors only once during init and not during each update call.

Would love to make a pull. Holler if needed!

**Comment by beaufortfrancois** [comment]
I've removed `updateDisplays` and `updateStorage` completely and it doesn't change anything as far as I can tell.
What we could do is to lower down the interval (500ms) but it makes the CPU usage UI kinda bland ;(

**Comment by vivekannan** [comment]
Yeah, i tried rewriting it so that the document.querySelector is run only
once, didn't make any difference either. :(
